.github/workflows/component-bumper.yml

```diff
@@ -13,6 +13,11 @@ jobs:
 
     - name: Check out code into the Go module directory
       uses: actions/checkout@v2
+      with:
+        ref: master
+
+    - name: Pull latest master
```

Comment:
Add a comment to remove this after everything is working fine.

Reply:
oh I may have not been clear - this is not supposed to be a temporary change.
This change will allow us to re-run the same job with the latest master (it doesn't rebase automatically like prow does)